books.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Remember that the `rentals` will be the link between the `Book` and `Person` classes in the `many to many` relationships. This means that you can only create a rental by having a date, book and person parameters when creating a `rental`. The `add_rental` function should create a new `rental` not push a single rental to the rentals array. You can read this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for a better understanding. You can also take a cue from the `add_rental` method in the `Person` class. 👍